x/distribution/keeper/genesis.go

```diff
@@ -123,8 +123,8 @@ func (k Keeper) InitGenesis(ctx context.Context, data types.GenesisState) error
 	}
 
 	balances := k.bankKeeper.GetAllBalances(ctx, moduleAcc.GetAddress())
-	if !balances.Equal(moduleHoldingsInt) {
-		return fmt.Errorf("distribution module balance does not match the module holdings: %s <-> %s", balances, moduleHoldingsInt)
+	if balances.IsAllLT(moduleHoldingsInt) {
```

Comment:
Can you test these conditions?

Reply:
As in test the failing case? Sure. Where should I add those tests?